.github/workflows/onRelease.yml

```diff
@@ -0,0 +1,21 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    container:
+      image: node:lts
+    steps:
+    - uses: actions/checkout@v2
+  npm:
+    runs-on: ubuntu-latest
+    container:
+      image: node:lts
+    steps:
+      - name: Set .npmrc
+        run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
```

Comment:
@CristiCanizales I missed this line - can you change this to ${{ secrets.NPM_TOKEN }}, and check that that resolves ok in your forked repo?

Reply:
actually... IDEE_NPM_TOKEN